projects/go-lib/src/lib/components/go-datepicker/calendar-views.scss

```diff
@@ -1,6 +1,7 @@
 @import '../../../styles/variables';
 
 .go-calendar__header {
+  color: $base-dark;
```

Comment:
These base colors are used to set up our theme colors. Should we use our theme color here instead? `$theme-light-color`

Reply:
@StevenUlmer Gotcha. So do we *always* want to reference the theme colors rather than the base colors?

Reply:
Almost always the answer to that is yes. There might be rare cases where we are doing something outside of the scope of a theme, but here this text should be the same as whatever the light theme is and if that themes color ever changes, then we want this to change as well.